website/docs/secrets/manage-secrets-gui.mdx

```diff
@@ -3,18 +3,131 @@ title: Manage Secrets GUI
+
+![Secrets Overview](./imgs/secretes-overview-1.png)
+
+## External Secrets (ESO)
+
+### Create External Secret CR (ESO)
+
+To create a new ExternalSecret CR, start by click on to the 'Create A Secret' button to navigate to the Creation page.
+
+![Create new Secret](./imgs/create-secret-1.png)
+
+Here, you will be prompted to enter the 'External Secret Name' and the 'Target K8s Secret Name'. Once you choose the 'Target Cluster', you will find a new list of all the 'Secret Stores' on this cluster to choose from.
+
+It's important to note that the chosen Secret Store may be a 'Cluster SecretStore' or a namespace scoped 'SecretStore'. 
+
+![Create new Secret](./imgs/create-secret-2.png)
+If you choose a namespace scoped SecretStore, the new secret will be created on the same namespace as the SecretStore.
+
+![Create new Secret](./imgs/create-secret-3.png)
+However, if you choose a ClusterSecretStore, you will have to enter the namespace in which the secret will be created.
+
+
+This process allows you to easily create new ExternalSecret CRs without needing to manually create them through YAML files or command line tools.
+
+### List External Secrets (ESO)
+
+![Secrets list](./imgs/secrets-overview-2.png)
+
+The List External Secrets section of the Manage Secrets GUI allows you to view all the external secrets that are currently stored in your Kubernetes cluster. This section provides an overview of each external secret, including its name, namespace, cluster, k8s-secret, secret-store and the Age. From this page, you can also navigate to the External Secret Details page to view more information about a specific secret.
+
+
+### External Secret Details (ESO)
+
+![External Secret Details](./imgs/eso-details-1.png)
+
+The External Secret Details section of the Manage Secrets GUI allows you to view the details of a specific external secret, including its name, namespace, data, and creation date. Below are the details that you can expect to see on this page:
+
+- **Status:** This indicates the current status of the external secret, which can be "Ready" or "Not Ready" depending on whether the external secret has been successfully created and is ready for use.
+
+- **Last Updated:** This shows the date and time when the external secret was last updated.
+
+- **External Secret:** This is the name of the external secret that you are viewing.
+
+- **K8s Secret:** This is the name of the Kubernetes secret that is associated with the external secret.
+
+- **Cluster:** This indicates whether the external secret is scoped to a specific namespace or to the entire cluster.
+
+- **Secret Store:** This shows the name of the secret store provider that is being used to store the external secret.
+
+- **Secret Store Type:** This indicates the type of secret store that is being used to store the external secret. In this case, the type is "AWS Secrets Manager".
+
+- **Secret path:** This is the path to the external secret within the cluster.
```

Comment:
I think it's the path on the operator, can you double check with @ahsayde or @waleedhammam?